### ISSUE TITLE
Some banned passwords are actually ROT-13 encoded.

### ISSUE THREAD

**Issue by goozak** [danpalmer/jquery.complexify.js#29]
Hello !
I was looking at the list of banned password and was wondering why some (like 'puvpntb') are there.  A web search found a [page](http://rixstep.com/1/1/20110720,00.shtml) that explained how Twitter had obfuscated its original list using ROT-13 (simple letters permutation) — which would actually make sense to prevent 'naughty' words triggering parental control software.

Since Complexify doesn't seem to 'decipher' this password list, I'm not sure the current list is completely achieving its goal.

**Comment by msigley** [comment]
Good catch Bruno. This regression is on me since I updated the password list. I'll look at decrypting them tomorrow and submit a pr.

-----Original Message-----
